CampaignConfig/mdc2020_DIOTail.ini

```diff
@@ -6,7 +6,7 @@ campaign_stage_list = DIOTail_fcl, DIOTail, DIOTaildigi_fcl, DIOTaildigi, DIOTai
 
 [campaign_defaults]
 vo_role=Production
-software_version=Offline
+software_version=mdc2020
```

Comment:
If you use MDC2020 you could create the desc field as software_version+release.   That assumes POMS lets you manipulate strings.

Reply:
I can change:
[sam_consumer]
appvers = %(release)s

However, I don't think I can put the following in *.ini file:
software_version=mdc2020%(release_v_o)s

This will require a user to explicitly set:
software_version=mdc2020ad
That becomes a nuisance to maintain.